Safety/boardfiles/Src/stm32f0xx_it.c

```diff
@@ -98,30 +99,30 @@ void HardFault_Handler(void)
 }
 
 /**
-  * @brief This function handles System service call via SWI instruction.
-  */
-void SVC_Handler(void)
-{
-  /* USER CODE BEGIN SVC_IRQn 0 */
+* @brief This function handles System service call via SWI instruction.
+*/
+// void SVC_Handler(void)
+// {
+//   /* USER CODE BEGIN SVC_IRQn 0 */
 
-  /* USER CODE END SVC_IRQn 0 */
-  /* USER CODE BEGIN SVC_IRQn 1 */
+//   /* USER CODE END SVC_IRQn 0 */
+//   /* USER CODE BEGIN SVC_IRQn 1 */
 
-  /* USER CODE END SVC_IRQn 1 */
-}
+//   /* USER CODE END SVC_IRQn 1 */
+// }
 
-/**
-  * @brief This function handles Pendable request for system service.
-  */
-void PendSV_Handler(void)
-{
-  /* USER CODE BEGIN PendSV_IRQn 0 */
+// /**
+// * @brief This function handles Pendable request for system service.
+// */
+// void PendSV_Handler(void)
+// {
+//   /* USER CODE BEGIN PendSV_IRQn 0 */
 
-  /* USER CODE END PendSV_IRQn 0 */
-  /* USER CODE BEGIN PendSV_IRQn 1 */
+//   /* USER CODE END PendSV_IRQn 0 */
```

Comment:
why are these commented out?

Reply:
They were causing build errors (redefined function) so they got the axe. There's something wrong with our CubeMX generation right now that was the source of our troubles for this PR. I think it should be a high priority item to address post-comp